network-mux/network-mux.cabal

```diff
@@ -81,8 +81,6 @@ test-suite test-network-mux
                        Test.Mux.ReqResp
   default-language:    Haskell2010
   build-depends:       base,
-                       typed-protocols,
-                       ouroboros-network-framework,
```

Comment:
This was so easy, because I untangled the dependency earlier on :) - at a cost of having yet another `req-resp` testing protocol.

Reply:
Yes, thank you! :-)